### ISSUE TITLE
[BUG] 'ExponentialSmoothing' object has no attribute 'historical_forecasts' when backtesting

### ISSUE THREAD

**Issue by baletercero** [unit8co/darts#267]
**Describe the bug**
When running an example from : https://medium.com/unit8-machine-learning-publication/darts-time-series-made-easy-in-python-5ac2947a8878 using AirPassengers.csv I get the above mentioned error , I can't find any documentation and am at a loss on why the backtesting doesn't work.

**To Reproduce**
import pandas as pd
from darts import TimeSeries

df = pd.read_csv('data/AirPassengers.csv', delimiter=",")
series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
train, val = series.split_after(pd.Timestamp('19580101'))

from darts.models import ExponentialSmoothing

model = ExponentialSmoothing()
model.fit(train)
prediction = model.predict(len(val))
import matplotlib.pyplot as plt

series.plot(label='actual')
prediction.plot(label='forecast', lw=2)
plt.legend()
plt.xlabel('Year')

from darts.models import Prophet
models = [ExponentialSmoothing(), Prophet()]
backtests = [model.historical_forecasts(series,
                            start=.5,
                            forecast_horizon=3)
             for model in models]

**Expected behavior**
I expected to be able to get a backtests object back and run : 
from darts.metrics import mape
series.plot(label='data')
for i, m in enumerate(models):
    err = mape(backtests[i], series)
    backtests[i].plot(lw=3, label='{}, MAPE={:.2f}%'.format(m, err))
plt.title('Backtests with 3-months forecast horizon')
plt.legend()

**System (please complete the following information):**
 - Python version: 3.8.5
 - darts version 0.6.0

**Additional context**

**Comment by hrzn** [comment]
Hi @baletercero ,
Normally the code from the article should work fine if you copy it verbatim. It is normal for the backtesting to take a while (a few minutes) though. If you get an error, could you provide the error message?
We had a small "outage" with the documentation today because we had a new release, but it's back online. You can find the forecasting models documentation here: https://unit8co.github.io/darts/generated_api/darts.models.forecasting_model.html

**Comment by camilaagw** [comment]
Hi @baletercero,
I tried to reproduce your error locally, but your code worked well on my MacBook Pro with the following settings:
Python version: 3.8.5
darts version: 0.6.0
To reproduce it, I created a virtual env: 
```
python3 -m venv darts-env
source darts-env/bin/activate
pip install 'u8darts[all]'
pip install ipykernel
python -m ipykernel install --user --name=darts-env
jupyter notebook
```
And then created a notebook with the kernel `darts-env` (instead of the usual `Python 3`)
Let us know if this works for you.
